packages/suite/src/components/suite/Settings/components/ActionColumn.tsx

```diff
@@ -19,7 +19,7 @@ export const ActionColumn = styled.div`
 
 const InputWrapper = styled.div`
     margin: 4px 0 4px 4px;
-    width: 170px;
+    width: 250px;
```

Comment:
Looks like a big change to me. We allow 16 chars, why to have such a big input. Error can be `Unsupported characters`, thats it

![Screenshot 2023-08-02 at 8 29 23](https://github.com/trezor/trezor-suite/assets/33235762/0c31e87b-28fe-41d4-ad58-ef8c0d924c78)